modules/bound-sa-tokens-configuring.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * authentication/bound-service-account-tokens.adoc
+
+[id="bound-sa-tokens-configuring_{context}"]
+= Configuring bound service account tokens using volume projection
+
+You can configure Pods to request bound service account tokens by using volume projection.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Set the service account issuer.
+
+.. Edit the `cluster` authentication object.
++
+----
+$ oc edit authentications cluster
+----
+
+.. Set the `spec.serviceAccountIssuer` field to the desired service account issuer value. The default is [x-]`https://kubernetes.default.svc`.
++
+[source,yaml]
+----
+spec:
+  serviceAccountIssuer: https://test.default.svc
+----
++
+// TODO: Does this have to start with `https://`?
++
+// TODO: does this value relate to the pod spec below at all?
+
+. Configure a Pod to use a bound service account token via volume projection.
+
+.. Create a file called `pod-projected-svc-token.yaml` with the following contents:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - image: nginx
+    name: nginx
+    volumeMounts:
+    - mountPath: /var/run/secrets/tokens
+      name: vault-token
+  serviceAccountName: build-robot
+  volumes:
+  - name: vault-token
+    projected:
+      sources:
+      - serviceAccountToken:
+          path: vault-token <1>
+          expirationSeconds: 7200 <2>
+          audience: vault <3>
+----
+<1> The path relative to the mount point of the file to project the token into.
+<2> Optionally set the expiration of the service account token, in seconds. The default is 3600 seconds (1 hour) and must be at least 600 seconds (10 minutes). The kubelet will start trying to rotate the token if the token is older than 80 percent of its time to live or if the token is older than 24 hours.
+<3> Optionally set the intended audience of the token. A recipient of a token must identify itself with an identifier specified in the audience of the token, and otherwise should reject the token. The audience defaults to the identifier of the API server.
++
+// TODO: Other callouts where they should make changes. serviceAccountName, others?
++
+// TODO: Is path required, and only audience and expiration are optional?
+
+.. Create the Pod:
++
+----
+$ oc create -f pod-projected-svc-token.yaml
+----
++
+The kubelet requests and stores the token on behalf of the Pod, makes the token available to the Pod at a configurable file path, and refreshes the token as it approaches expiration.
++
+////
+TODO: Error trying instructions as is:
+
+Error from server (Forbidden): error when creating "pod-projected-svc-token.yaml": pods "nginx" is forbidden: error looking up service account default/build-robot: serviceaccount "build-robot" not found
```

Comment:
The service account does need to exist. The upstream docs gloss over that, but it's probably worth mentioning here (at least 'service account must be created first' if not actual instructions for creation).

Reply:
Added it as a prereq, and gave it a callout in the yaml.